src/particle.cpp

```diff
@@ -286,6 +287,8 @@ void Particle::event_collide()
     }
   }
 
+  if (!model::active_pulse_height_tallies.empty() && type() == ParticleType::photon){pht_collision_energy();}
```

Comment:
Starting to go through this PR for a full review but one quick request -- it looks like a number of the files here have not been run through clang-format, which we use for automatic formatting of C++ source files. Can you take a look at the guidelines [here](https://docs.openmc.org/en/latest/devguide/styleguide.html#automatic-formatting) on automatic formatting?

Reply:
Yes! All the changed files are now in clang format. As you'll see, this also changed a few other lines (not part of this pull request) that weren't in clang format yet.... I hope this is not a problem.